packages/spektate/src/Deployment.ts

```diff
@@ -387,15 +391,22 @@ export class Deployment {
     return "Incomplete";
   }
 
-  public fetchAuthor(callback: (author: IAuthor | undefined) => void): void {
-    if (this.srcToDockerBuild && this.srcToDockerBuild.repository) {
-      this.srcToDockerBuild.repository
-        .getAuthor(this.srcToDockerBuild.sourceVersion)
-        .then((author: IAuthor | undefined) => {
-          this.author = author;
-          callback(author);
-        });
-    }
+  public fetchAuthor(): Promise<IAuthor | undefined> {
```

Comment:
nit: We can begin adding summary/documentation to functions we add/change. Thoughts?